KarhooUISDK/Translations/it.lproj/Localizable.strings

```diff
@@ -618,9 +620,9 @@
 
 "Text.QuoteCategory.Electric" = "Elettrico";
 
-"Text.QuoteCategory.Exec" = "Esecutivo";
+"Text.QuoteCategory.Exec" = "Executive";
 
-"Text.QuoteCategory.Executive" = "Esecutivo";
+"Text.QuoteCategory.Executive" = "Executive";
```

Comment:
looks like regression. @olivierkarhoo do you know anything about this change?

Reply:
Yes executive is correct, I changed it after asking to Italian native person yesterday

Reply:
@dianakarh